src/components/dialog/dialog.component.tsx

```diff
@@ -91,6 +91,8 @@ export interface DialogProps extends ModalProps, TagProps {
   role?: string;
   /** Padding to be set on the Dialog content */
   contentPadding?: ContentPaddingInterface;
+  /** Allows developers to change the background color of the content to grey */
+  hasGreyBackground?: boolean;
```

Comment:
**suggestion(non-blocking)**: amending prop name to exclude token value

Given we have no control over the token values from the `@sage/design-tokens` package, there is potential that the `colorsUtilityMajor025` token could be updated in the future to be no longer grey. So I would amend the prop name slightly to exclude token value, to avoid the need to update the prop name later as a breaking change.

```suggestion
  /** Change the background color of the content to old grey color */
  legacyBackgroundColor?: boolean;
```